packages/node/src/policy-gen/policy-gen-util.js

```diff
@@ -5,62 +5,41 @@
  * @internal
  */
 
-import {
-  ATTENUATORS_COMPARTMENT,
-  LAVAMOAT_PKG_POLICY_ROOT,
-} from '../constants.js'
-import { hasValue, isObject } from '../util.js'
+import { LAVAMOAT_PKG_POLICY_ROOT } from '../constants.js'
+import { hasValue, isObjectyObject } from '../util.js'
 
 /**
  * @import {CompartmentDescriptor} from '@endo/compartment-mapper'
  * @import {ModuleDescriptor} from 'ses'
+ * @import {CanonicalName} from '../internal.js'
  */
 
 /**
  * Determine the canonical name for a compartment descriptor
  *
  * @param {CompartmentDescriptor} compartment Compartment descriptor
- * @returns {string} Canonical name
- * @throws {TypeError} If compartment has no path
+ * @param {boolean} [trustRoot=true] If `false`, never return a canonical name
+ *   of {@link LAVAMOAT_PKG_POLICY_ROOT}. Default is `true`
+ * @returns {CanonicalName} Canonical name
+ * @throws {ReferenceError} If compartment has no path
  * @internal
  */
-export const getCanonicalName = (compartment) => {
+export const getCanonicalName = (compartment, trustRoot = true) => {
   // NOTE: the algorithm creating paths happens to be identical to the one in @lavamoat/aa package. Not that it matters because policies cannot be reused between this and other lavamoat tools.
-  if (compartment.name === ATTENUATORS_COMPARTMENT) {
-    return ATTENUATORS_COMPARTMENT
-  }
   if (!compartment.path) {
     throw new ReferenceError(
       `Computing canonical name failed: compartment "${compartment.name}" (${compartment.location}) has no "path" property; this is a bug`
     )
   }
   if (compartment.path.length === 0) {
```

Comment:
- [ ] create a function that accepts `Compartment` and returns `true` if the `Compartment` is the entry